src/components/Theme/index.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { createContext, useContext, useEffect, useState } from 'react'
+import { ThemeProvider as StyledComponentsThemeProvider } from 'styled-components'
+import { DarkTheme, LightTheme } from '../../themes'
+
+export enum Theme {
```

Comment:
`Enum` is a TypeScript feature that defines a set of named constant, right? What is the advantage of using it here instead of just defining two separate constants?

Reply:
One of the advantages is a hint of all possible values

Reply:
Yes, `Enum` is TS feature, that comes from other typed languages. It's very useful for keeping programmatic related constant values together as the same type. More about them in the official documentation - https://www.typescriptlang.org/docs/handbook/enums.html :wink: